### DISCUSSION TITLE
Mqtt Publish jaon data in rule

### DISCUSSION THREAD

**Discussion by usrrsr** [arendst/Tasmota#18679] [Q&A]
Hi

I am trying following rule in console but its shows error

Rule2+ON power5#State=1 do backlog Publish stat/246F284/Notification  json.dump({ "lpm":10,"Count":15}) endon

Or also tried 

Rule2+ON power5#State=1 do backlog Publish stat/246F284/Notification {"Switch2":"ON"} ENDON

Let me know my mistake what i am doing wrong here

**Comment by barbudor** [comment]
As you don't give the error it is supposed to report it is not easy

However I feel you are missing a space between `Rule2` and the rest

In the 1st rule, if you are expecting `json.dump({ "lpm":10,"Count":15})` to actually do something, it will not. That would just publish it as a string.

**Reply by sfromis** [reply]
Beware that after saving the rule, you also have to enable it, using another invocation of the [Rule](https://tasmota.github.io/docs/Commands/#rule) command.
